package.json

```diff
@@ -28,7 +28,7 @@
     "test:unit": "vue-cli-service test:unit",
     "test:unit:snapshot": "vue-cli-service test:unit -u",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test:unit:ci": "npm run test:unit -- --ci --coverage",
+    "test:unit:ci": "npm run test:unit -- --ci --coverage && open coverage/lcov-report/index.html",
```

Comment:
This is not needed on CI (Travis) - nobody can see it. :) The command is targeting  specifically CI env (`test:unit:ci`).  You can create another command to cover running it locally.